test/jdk/java/net/httpclient/MaxStreams.java

```diff
@@ -220,18 +214,14 @@ public void handle(Http2TestExchange t) throws IOException {
                         getLatch().await();
                         System.err.println("Latch resume");
                     } catch (InterruptedException ee) {}
+                } else if (c == MAX_STREAMS + 1) {
+                    // client issues MAX_STREAMS + 3 requests in total
+                    // but server should only see MAX_STREAMS + 2 in total. One is rejected by client
+                    // counter c captured before increment so final value is MAX_STREAMS + 1
```

Comment:
Can we keep/add a trace here to say that counter is reset to 0?
Otherwise look good.

Reply:
Log added.